README.md

```diff
@@ -73,7 +73,7 @@ All images are based on their official variants, being:
 ## Alpine Linux situation
 
 Even though both of the images are based on the Alpine Linux, the PHP official repository gives us the option to choose
-between its versions, at this moment being `3.8` or `3.9`.
+between its versions, at this moment being `3.7`, `3.8`, `3.9` or `3.10`.
```

Comment:
i didn't really understand this sentence. have i corrected it properly?

Reply:
Yes!
The intent is to say you can choose between alpine versions

Reply:
According to the [current documentation](https://github.com/docker-library/docs/blob/master/php/README.md#supported-tags-and-respective-dockerfile-links), `3.7` and `3.8` are no longer supported. Can we drop them as well?